app/controllers/api/v1/reservations_controller.rb

```diff
@@ -2,7 +2,8 @@ class Api::V1::ReservationsController < ApplicationController
   before_action :find_reservation, only: %i[show update destroy]
   # GET /reservations by user
   def index
-    @reservations = Reservation.all
+    @current_user = User.find(request.params['user_id'])
```

Comment:
Great job so far!!!

This is OPTIONAL: `current_user` is already a name from Devise, so you can consider use another like @user, or other.